internal/generator/vector/output/loki/loki_conf_test.go

```diff
@@ -181,6 +182,96 @@ strategy = "basic"
 user = "username"
 password = "password"
 
+`,
+		}),
+		Entry("with bearer token", generator.ConfGenerateTest{
+			CLFSpec: logging.ClusterLogForwarderSpec{
+				Outputs: []logging.OutputSpec{
+					{
+						Type: logging.OutputTypeLoki,
+						Name: "loki-receiver",
+						URL:  "https://logs-us-west1.grafana.net",
```

Comment:
if this is a live endpoint, is a good idea to rely on it in our tests?
if it's not (dig is not resolving it for me), can we stick to example.com?

Reply:
this is not a live endpoint, just for unit testing generating configuration

Reply:
can we change it to to example.com though so the intent is clear?